backend/.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Prettier's default config is great when starting a new project, however when introducing prettier into an existing project, I think it's best to stick to the project's standards in order to reduce the impact. A lower impact means it's easier to navigate the versioning history down the line. It also eases reverting or merging concurrent changes.

For this project, I suggest turning `arrowParens` to `always`, `singleQuote` to `true`, and `trailingComma` to `all`.